components/director/internal/domain/application/resolver_test.go

```diff
@@ -295,14 +295,16 @@ func TestResolver_DeleteApplication(t *testing.T) {
 	modelApplication := fixModelApplication("foo", "tenant-foo", "Foo", "Bar")
 	gqlApplication := fixGQLApplication("foo", "Foo", "Bar")
 	testErr := errors.New("Test error")
-
+	testAuth := fixOauth()
 	txGen := txtest.NewTransactionContextGenerator(testErr)
 
 	testCases := []struct {
```

Comment:
I think it would be nice to add success case where `SysAuthSvc` returns 3 auths (one with nil value, one with OAuth credentials and one with Basic credentials).